packages/codec/lib/interface/decoders/contract.ts

```diff
@@ -25,6 +25,10 @@ import { isWordsLength, equalSlots } from "../../utils/storage";
 import { StoragePointer } from "../../types/pointer";
 import { ContractBeingDecodedHasNoNodeError, ContractAllocationFailedError } from "../../interface/errors";
 
+/**
+ * The ContractDecoder class.  Spawns the ContractInstanceDecoder class.
```

Comment:
ContractInstanceDecoder should be link, possibly with link to specific method that gives the user the ContractInstanceDecoder

Reply:
Is there an easy way to do this sort of link from within TypeDoc?  Like, allowing one to just link to a method or class, without needing to know in advance the full URL?

Reply:
Oh, yup, there is.  Yay!